data/manual-config.json

```diff
@@ -1,3 +1,12 @@
 {
-  "resources": []
-}
+  "resources": [
+    {
+      "type": "github",
+      "username": "chrisgrieser",
+      "repo": ".config",
+      "tags": [
+        ""
```

Comment:
Did this happen through the CLI?  If so I'll need to fix, but in the meantime, could you please remove the element?  Thanks!

Reply:
yes, it was created by `make resource`, following exactly the instructions…